### PR TITLE
Use real client ip if proxy is being used

### PR DESCRIPTION
**Pull request by unho**
Fixes issue reported by @chazotte

### CODE REVIEW

File: pootle/apps/contact/forms.py [hunk 1]
```diff
@@ -52,6 +52,9 @@ def get_context(self):
         """Get context to render the templates for email subject and body."""
         ctx = super(ContactForm, self).get_context()
         ctx['server_name'] = settings.POOTLE_TITLE
+        ctx['ip_address'] = (self.request.META.get('HTTP_X_FORWARDED_FOR')
+                             if 'HTTP_X_FORWARDED_FOR' in self.request.META
+                             else self.request.META.get('REMOTE_ADDR'))
```
Comment (phlax) [inline pootle/apps/contact/forms.py]: how about:

```
self.request.META.get("HTTP_X_FORWARDED_FOR", self.request.META.get("REMOTE_ADDR"))
```

Comment (unho) [inline pootle/apps/contact/forms.py]: That works too, but found it to be too long.

Comment (phlax) [inline pootle/apps/contact/forms.py]: its much easier to read

**Comment by phlax** [comment]
lgtm

**Pull request merged**